src/main/java/org/opentripplanner/updater/stoptime/GtfsRealtimeFileTripUpdateSource.java

```diff
@@ -42,7 +42,15 @@
      * previous updates should be disregarded
      */
     private boolean fullDataset = true;
-    
+
+    /**
+     * True iff updates from this realtime feed should always be considered incremental,
```

Comment:
"iff" with two F is logician shorthand for "if and only if". https://en.wikipedia.org/wiki/If_and_only_if